charts/nm-vllm-grafana-dashboards/templates/config-map-list.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMapList
+items:
+{{- $files := .Files.Glob "dashboards/*.json" }}
+{{- range $path, $fileContents := $files }}
+{{- $dashboardName := regexReplaceAll "(^.*/)(.*)\\.json$" $path "${2}" }}
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: {{ template "nm-vllm-grafana-dashboards.fullname" $ }}-grafana-dashboard
+    labels:
+      {{- if $.Values.label }}
+      {{ $.Values.label }}: {{ ternary $.Values.labelValue "1" (not (empty $.Values.labelValue)) | quote }}
+      {{- end }}
```

I think we should do what the [`grafana`](https://github.com/grafana/helm-charts/blob/main/charts/grafana/templates/dashboards-json-configmap.yaml#L13) chart does here instead. It's simpler and not obviously problematic. I would also like our defaults to match the `grafana` default (`""`), and I think this would work with `""` given that it seems to work for grafana
```suggestion
      {{ $.Values.label }}: {{ $.Values.labelValue | quote }}
```

---

I did this because I wanted to test the value and that meant replacing "1"